src/experimental/standalone_pem/BUILD.bazel

```diff
@@ -57,6 +57,7 @@ pl_cc_library(
 pl_cc_binary(
```

Comment:
Did you consider linking a different libc like musl? I believe static linking glibc is generally [discouraged](https://stackoverflow.com/questions/57476533/why-is-statically-linking-glibc-discouraged), although I don't have experience with these problems.